FAQ_CN.md

```diff
@@ -56,3 +56,16 @@ APISIX 是当前性能最好的 API 网关，单核 QPS 达到 2.3 万，平均
 5. 高性能
 
 APISIX 需要一个配置中心，上面提到的很多功能是传统关系型数据库和KV数据库是无法提供的。与 etcd 同类软件还有 Consul、ZooKeeper等，更详细比较可以参考这里：[etcd why](https://github.com/etcd-io/etcd/blob/master/Documentation/learning/why.md#comparison-chart)，在将来也许会支持其他配置存储方案。
+
+## 为什么在用 Luarocks 安装 APISIX 依赖时会遇到超时，很慢或者不成功的情况？
```

Comment:
you can also use docker to install APISIX.

Reply:
yes, but docker isn't the only way, this choice is too much subjectivity